app/panel/components/Panel.jsx

```diff
@@ -275,6 +295,8 @@ class Panel extends React.Component {
 
 		sendMessage('promoModals.sawInsightsPromo', '', 'metrics');
```

Comment:
what does the 'metrics' argument do here?  looks like the other message calls use the format `sendMessage('promoModals.sawPlusPromo', {});`

Reply:
Forgot to ask Ben to change this when we did the Insights promo modal PR. Removing.